### DISCUSSION TITLE
[不具合] Mac Safari で マイページが表示されないようです。

### DISCUSSION THREAD

**Discussion by ttatsf** [increments/qiita-discussions#614] [要望・アイデア・不具合]
最近気付いたのですが、Safariでマイページが開くと空白になるようです。

<img width="1281" alt="スクリーンショット 2024-02-09 17 42 14" src="https://github.com/increments/qiita-discussions/assets/4555650/82ce810a-6ead-426e-a721-0b95332965dd">

Chromeだと大丈夫でした。

**Comment by tanaka-asami415** [comment]
@ttatsf 
ご報告ありがとうございます。
一度キャッシュクリアをお試しいただけますでしょうか🙏

解決しない場合は、OSとブラウザのバージョンを教えていただけると幸いです🙇

**Comment by ttatsf** [comment]
キャッシュクリアしても変りませんでした。

<img width="587" alt="スクリーンショット 2024-02-10 8 41 00" src="https://github.com/increments/qiita-discussions/assets/4555650/0c0f1358-1012-4b4a-b596-f7add486ee18">
<img width="289" alt="スクリーンショット 2024-02-10 8 39 48" src="https://github.com/increments/qiita-discussions/assets/4555650/d8cf864b-a476-49c8-8c75-72d718c63d64">

**Comment by JunichiIto** [comment]
僕のMac Safariは表示されますね。
![Screenshot 2024-02-10 at 9 05 11](https://github.com/increments/qiita-discussions/assets/1148320/0e5d3ddf-cc7b-4f32-9897-db42f3189ddf)
ttatsfさんのページも表示されます。
![Screenshot 2024-02-10 at 9 06 00](https://github.com/increments/qiita-discussions/assets/1148320/3093fecc-1394-41eb-8a68-c43a7dee5a26)

環境は以下の通りです。

- Safari Version 17.2.1 
- macOS 14.2.1 

広告ブロッカーが必要なJS読み込みを妨害してたりしませんかね？
プライベートウインドウで開くとどうでしょう？

あと、[Safari 14.1.2は2021年リリース](https://ja.wikipedia.org/wiki/Safari)なのでブラウザが古いのが原因、ということもあるかもしれません。

**Comment by ttatsf** [comment]
@JunichiIto そうですか...特にブロッカーはいれてないんですけどね...
プライベートウインドウで開くといけましたが、そこからログインするとまた空になってしまいました。
<img width="675" alt="スクリーンショット 2024-02-10 9 42 35" src="https://github.com/increments/qiita-discussions/assets/4555650/d78795dc-9f23-4fe8-810c-8e0805e37451">


<img width="644" alt="スクリーンショット 2024-02-10 9 43 33" src="https://github.com/increments/qiita-discussions/assets/4555650/b4169f24-1d39-4579-88bc-3a0a5e2bdc11">

**Reply by JunichiIto** [reply]
なるほどー。開発者ツールなどを開くとJSエラーが出てたりするかもしれませんね。
ただ、前回もコメントしたとおりブラウザが古いのも怪しいので、まずは最新のSafariでも問題が再現するかどうかを確認した方が良いかもしれません。

[対応ブラウザ \| Qiita ヘルプ](https://help.qiita.com/ja/articles/supported-browsers) を見ると、Safariのサポートバージョンは "Last 2 versions" だそうです。
ということは、Safari 16と17だけがサポート対象なのかな？という気がします。
公式の見解はQiitaスタッフの方からの返信を待つ必要がありますが、とりあえずご参考までに。

**Reply by ttatsf** [reply]
新しめの方{Monterey)でみたらちゃんと見れました。エラーも出てませんでした。やっぱ古いからですかね?
ちなみに出ていたエラーはこんな感じです。：

<img width="1439" alt="スクリーンショット 2024-02-10 10 26 18" src="https://github.com/increments/qiita-discussions/assets/4555650/c2f1ca41-7469-4d42-91c4-59d25cbc2435">

おりたたみの詳細：

<img width="470" alt="スクリーンショット 2024-02-10 10 26 59" src="https://github.com/increments/qiita-discussions/assets/4555650/2ad9c000-be97-4175-8c1d-2093ef420562">

**Reply by JunichiIto** [reply]
> 新しめの方{Monterey)でみたらちゃんと見れました。エラーも出てませんでした。やっぱ古いからですかね?

JSエラーの内容まではわかりませんが、状況から察するにその可能性が高そうですね。

**Comment by tanaka-asami415** [comment]
@ttatsf 
キャッシュクリアなどご対応いただきありがとうございます🙏

JunichiIto さんにも記載いただいていますが、ブラウザのバージョンが古いことと（[対応ブラウザ](https://help.qiita.com/ja/articles/supported-browsers)）、OSも古いことが原因の可能性が高いです。
お手数ですが、Safariで閲覧されたい場合はバージョンを上げていただけると幸いです🙇

@JunichiIto 
回答いただきありがとうございます🙏✨
